lib/lens.rb

```diff
@@ -22,5 +24,17 @@ def configure
     def configuration
       @configuration ||= Configuration.new
     end
+
+    def start
+      Worker.start(configuration)
+    end
+
+    def stop
+      Worker.stop
+    end
+  end
+
+  if defined?(Rails)
+    require "lens/railtie"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.